lib/components/blockquote.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/forbid-prop-types */
+
+import React from 'react';
+
+const setup = ({ blockList }) => {
+  const Blockquote = ({ items, pullQuote = false }) =>
+    <blockquote className={pullQuote ? 'q' : null}>{blockList(items)}</blockquote>;
```

Comment:
because isn't a blockquote just a big quote? why should we be able to put all the article elements (including itself) into it when we can't put them all into the other elements?